development.ini.sample

```diff
@@ -135,6 +138,11 @@ meetings.timezone = America/Denver
 meetings.default_duration = 7200
 meetings.icalendar.prodid = -//Mines ACM//web//EN
 
+# Custom Assets Configuration
+custom_assets.dir = /
+custom_assets.css = custom.css
+custom_assets.logo = custom.png
```

Comment:
I think it's a good idea to just have the paths be equivalent to the logos under public, rather than configuring the file name. This allows the sysadmin the guaruntee that the only overridden assets will have the same paths as the public directory.

So these keys should not be necessary.